cli/tauri.js/src/types/config.schema.json

```diff
@@ -399,6 +399,9 @@
             "active": {
               "description": "whether we should use the embedded-server or the no-server mode",
               "type": "boolean"
+            },
+            "port": {
+              "type": "string"
```

Comment:
I think we should add a description, because it should be clear that this is a fixed port.

Reply:
at the rust runner code, it do compare if port == "random", unless changing that too..

Reply:
yeah just add a JSDoc to the type definition like some of the other types in there